igcollect/logfile_values.py

```diff
@@ -169,13 +168,14 @@ def parse_args():
 
 def get_metrics_values(line, metrics, time_format, columns_num, time_column):
```

Comment:
Semantically the method name is misleading, should rather be `fetch` or `insert` or the like, as it doesn't return the values but insert them into `metrics` parameter.

Reply:
Resolve as it is out of scope for this PR.